docker-compose.yml

```diff
@@ -666,6 +666,36 @@ services:
         /arrow/ci/scripts/python_build.sh /arrow /build &&
         /arrow/ci/scripts/integration_turbodbc.sh /turbodbc /build"]
 
+  conda-python-kartothek:
+    # Possible $TURBODBC parameters:
+    #  - `latest`: latest release
+    #  - `master`: git master branch, use `docker-compose run --no-cache`
+    #  - `<version>`: specific version available under github releases
+    # Usage:
+    #   docker-compose build conda-cpp
+    #   docker-compose build conda-python
+    #   docker-compose build conda-python-kartothek
+    #   docker-compose run --rm conda-python-kartothek
+    image: ${REPO}:${ARCH}-conda-python-${PYTHON}-kartothek-${TURBODBC}
```

Comment:
what exactly is the turbodbc coupling here?

Reply:
Copy-paste failure by me.